pkgs/tools/admin/syft/default.nix

```diff
@@ -23,6 +23,8 @@ buildGoModule rec {
   vendorSha256 = "sha256-nb7QcdmwAfYDTzCFNjs7uKwK/gng2iMD36ANaFSsftk=";
 
   nativeBuildInputs = [ installShellFiles ];
+  buildInputs = lib.optionals (stdenv.isDarwin && stdenv.isx86_64)
```

Comment:
Need to add `stdenv` somewhere after `lib` in the function args at the top.

Reply:
Yeah the whole thing doesnt work anyway. Waiting on

#101229

Ill just amend this PR to mark it broken on x86_64 darwin